lib/intercom_flutter.dart

```diff
@@ -28,6 +28,10 @@ class Intercom {
     return _unreadChannel.receiveBroadcastStream();
   }
 
+  static Future<dynamic> setBottomPadding(int padding) {
```

Comment:
Could you add the docs for this new method, please? Also, what happens if `padding` is `null`? Is it a valid value for the native SDK? I see it's avoided on Android, but not on iOS

Reply:
Thank you for thereview. I implemented these wanted changes to new commit.

Reply:
Thanks!